docs/editors/vim.md

```diff
@@ -165,20 +165,30 @@ inoremap <silent><expr> <cr> pumvisible() ? coc#_select_confirm()
 
 ### Installing coc-metals
 
-Once you have `coc.nvim` installed, you can then install Metals by running.
+Once you have `coc.nvim` installed, you can then install Metals a few different ways. The easiest is
+by running:
 
 ```vim
 :CocInstall coc-metals
 ```
-If you'd like to use the latest changes on master, you can also just build from source by using a plugin
-manager to download the extension. If you do this and you've had `coc-metals` installed before with `:CocInstall`,
-make sure you run `:CocUninstall coc-metals` to remove it. Then, if you are using [`vim-plug`](https://github.com/junegunn/vim-plug)
+
+If you'd like to use the latest changes on master, you can also use the url of the repo in the
+command like so:
+
+```vim
+:CocInstall https://github.com/ckipp01/coc-metals
+```
+If you'd like to use the latest changes on master, but would prefer managing the plugin using a plugin
+manager to download the extension, make sure you run `:CocUninstall coc-metals` if you've had
```

Comment:
>  make sure you run `:CocUninstall coc-metals`

This is a bit hidden inside the paragraph. It might be good to extract it into a code fence like this
````
```
:CocUninstall coc-metals
```
````